File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int l=0;
        for(int r=1;r<nums.size();r++)
        {
        if(nums[l]<nums[r])
        {
        nums[l+1]=nums[r];
        l++;
        }
       
        }
        return l+1;
    }
};

File: 26-remove-duplicates-from-sorted-array/remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        s=set(nums)
        s=list(s)
        s=sorted(s)
        for i in range(len(s)):
            nums[i]=s[i]
        return len(s)